From: Eugeni Dodonov <eugeni.dodonov@intel.com>
Subject: [PATCH 1/1] drm/i915: add Ivy Bridge GT2 Server entries
Date: Thu, 29 Mar 2012 20:55:48 -0300

This adds PCI ID for IVB GT2 server variant which we were missing.

Signed-off-by: Eugeni Dodonov <eugeni.dodonov@intel.com>
[rediffed-for-3.3.1 / tmb]
Signed-off-by: Thomas Backlund <tmb@mageia.org>

---
 drivers/char/agp/intel-agp.h    |    1 +
 drivers/char/agp/intel-gtt.c    |    2 ++
 drivers/gpu/drm/i915/i915_drv.c |    1 +
 3 files changed, 4 insertions(+)

--- linux/drivers/char/agp/intel-agp.h.orig	2012-04-03 00:01:14.000000000 +0300
+++ linux/drivers/char/agp/intel-agp.h	2012-04-03 02:15:00.356263751 +0300
@@ -235,6 +235,7 @@
 #define PCI_DEVICE_ID_INTEL_IVYBRIDGE_M_GT2_IG		0x0166
 #define PCI_DEVICE_ID_INTEL_IVYBRIDGE_S_HB		0x0158  /* Server */
 #define PCI_DEVICE_ID_INTEL_IVYBRIDGE_S_GT1_IG		0x015A
+#define PCI_DEVICE_ID_INTEL_IVYBRIDGE_S_GT2_IG		0x016A
 
 int intel_gmch_probe(struct pci_dev *pdev,
 			       struct agp_bridge_data *bridge);
--- linux/drivers/char/agp/intel-gtt.c.orig	2012-04-03 00:01:14.000000000 +0300
+++ linux/drivers/char/agp/intel-gtt.c	2012-04-03 02:15:49.254661149 +0300
@@ -1461,6 +1461,8 @@ static const struct intel_gtt_driver_des
 	    "Ivybridge", &sandybridge_gtt_driver },
 	{ PCI_DEVICE_ID_INTEL_IVYBRIDGE_S_GT1_IG,
 	    "Ivybridge", &sandybridge_gtt_driver },
+	{ PCI_DEVICE_ID_INTEL_IVYBRIDGE_S_GT2_IG,
+	    "Ivybridge", &sandybridge_gtt_driver },
 	{ 0, NULL, NULL }
 };
 
--- linux/drivers/gpu/drm/i915/i915_drv.c.orig	2012-04-03 00:01:14.212837894 +0300
+++ linux/drivers/gpu/drm/i915/i915_drv.c	2012-04-03 02:13:32.179542129 +0300
@@ -283,6 +283,7 @@ static const struct pci_device_id pciidl
 	INTEL_VGA_DEVICE(0x0152, &intel_ivybridge_d_info), /* GT1 desktop */
 	INTEL_VGA_DEVICE(0x0162, &intel_ivybridge_d_info), /* GT2 desktop */
 	INTEL_VGA_DEVICE(0x015a, &intel_ivybridge_d_info), /* GT1 server */
+	INTEL_VGA_DEVICE(0x016a, &intel_ivybridge_d_info), /* GT2 server */
 	{0, 0, 0}
 };
 
